src/dialect/duckdb.rs

```diff
@@ -66,4 +66,9 @@ impl Dialect for DuckDbDialect {
     fn supports_explain_with_utility_options(&self) -> bool {
         true
     }
+
+    /// See DuckDB <https://duckdb.org/docs/sql/statements/load_and_install.html#load>
+    fn supports_load_extension(&self) -> bool {
```

Comment:
It looks like the Generic dialect should also support this one per the previous behavior?

Reply:
Previously, the Generic dialect did support this behavior, but I couldn't find the keyword "load" in ISO/IEC 9075. Therefore, I'm unsure whether we should continue supporting this behavior in the Generic dialect. @iffyio What is your opinion on this?